carapace-server/src/main/java/org/carapaceproxy/core/RuntimeServerConfiguration.java

```diff
@@ -77,7 +78,10 @@ public class RuntimeServerConfiguration {
     private boolean cacheDisabledForSecureRequestsWithoutPublic = false;
     private String mapperClassname;
     private String accessLogPath = "access.log";
+
     private String accessLogTimestampFormat = "yyyy-MM-dd HH:mm:ss.SSS";
+    private DateTimeFormatter accessLogTimestampFormatter;
```

Comment:
Maybe if this is used by requestslogger only, it should be inside RequestsLogger

Reply:
Make sense